Main.java

```diff
@@ -0,0 +1,66 @@
+package com.company;
+
+import java.io.*;
+import java.util.StringTokenizer;
+
+public class Main {
+
+    public static void main(String[] args) {
+
+        if (args.length != 2) {
+            System.out.println("File path and search word cannot be empty");
+            return;
+        }
+        String textFilePath = args[0];
+        String searchKeyword = args[1];
+        File file = new File(textFilePath);
+        System.out.println("Processing...");
+        validateFile(file);

```

Comment:
Why is this method called here?

Reply:
It was there by mistake. I have removed it.